package.json

```diff
@@ -22,7 +22,7 @@
     ]
   },
   "dependencies": {
-    "@polkadot/api": "^1.0.0-beta.15"
+    "@polkadot/api": "1.0.0-beta.15"
```

Comment:
this is also weird. Something's wrong.. it added `.18` to the `yarn.lock`, but not the package.json..?

Reply:
So there are two "@polkadot/api": one in the root package.json, another in back/node_watcher/package.json. You'd need to :
- either remove "@polkadot/api" from one of the 2 places (i'd say from the root one)
- or `yarn add` in both places

probably the 1st one is simpler, so that in the future we only have one place to update "@polkadot/api"

Reply:
That makes sense. The front needs the package so I'll remove it from the back and upgrade again, that should solve it.